package.json

```diff
@@ -37,6 +37,7 @@
     "watch-build": "rollup -cw",
     "format": "prettier --write .",
     "check-format": "prettier --check .",
+    "check-changeset": "changeset status --since=origin/main",
```

Comment:
Why do we need the `--since=origin/main`? In other repos, I haven't needed that.

Reply:
I added it following the suggestion in their documentation ([here](https://github.com/changesets/changesets/blob/main/docs/automating-changesets.md#blocking)).
The way I understand it is that it won't just check that a changeset is present, but it will check that a new changeset is present w.r.t. the ones that are already accumulated in origin/main.